### PR TITLE
Update openshift-operatorhub-kubecost-install.md

### PR DESCRIPTION
**Pull request by linhlam-kc**
Update example CRD templates to deploy Kubecost v1.99 that supports OCP 12.0 (K8s 1.25)

### CODE REVIEW

File: openshift-operatorhub-kubecost-install.md [hunk 1]
```diff
@@ -63,77 +63,202 @@ kubectl apply -f example-crd.yaml -n kubecost
...
+  kubecostNetworkCosts:
+    securityContext: {}
+    containerSecurityContext: {}
+
   prometheus:
     nodeExporter:
       enabled: false
-    serviceAccounts:
-      nodeExporter:
-        create: false
```
Comment (srpomeroy) [inline openshift-operatorhub-kubecost-install.md:87]: @linhlam-kc Won't removing the service account block cause the node exporter service account to be created?

Comment (linhlam-kc) [reply openshift-operatorhub-kubecost-install.md:87]: @srpomeroy Disabling nodeExporter should be enough according to this [condition](https://github.com/kubecost/openshift-helm-chart/blob/bc0e157f8d9376f93aaebe60ae300aaa0305c21a/cost-analyzer/charts/prometheus/templates/node-exporter-serviceaccount.yaml#L2) I do not see it is created in my testing last week as well on the OCP cluster. Let me try it real quick again.

Comment (linhlam-kc) [reply openshift-operatorhub-kubecost-install.md:87]: I can confirm that disable promethus.nodeExporter.enabled will also disable the creation of nodeExporter serviceAccount

Comment (srpomeroy) [reply openshift-operatorhub-kubecost-install.md:87]: Awesome! There must have been a change. That was not the case previously.

**Review by srpomeroy — APPROVED** [review]
LGTM

**Pull request merged**